### DISCUSSION TITLE
General questions on using UMEP for UWG processing

### DISCUSSION THREAD

**Discussion by PetitMote** [UMEP-dev/UMEP#410] [Q&A]
Hi guys,

I’m "playing" a bit with UMEP and UWG recently. Our goal is to produce a reference map of the UHI on my city. I’m starting to understand how it all works. However, some parts of the pre-processors are still obscure to me, and I think I need some help to understand them.

I have difficulties mainly with the Land Cover Fraction and the Morphometric Calculator. What parameters should I put there?

- Land Cover Fraction: I think I should tick the *Search throughout the grid extent*. I believe UWG takes in the cover fractions only of the grid cell? Is the wind direction relevant?
- Morphometric Calculator: Should I search throughout the grid extent, or from the grid centroid and search beyond the grid cell? Does it depend of the size of the grid? Is there a roughness calculation method more relevant than the others?

Also, until now, I’ve been using grid cells of 100m*100m. Is it possible to reduce the size of the cells, and to have a grid with more precision?

Finally, how doest the UWG Analyzer works? I think it’s calculating the difference between the reference meteo file, and the simulated meteo file, is that it? Is it differentiating night and day?

By the way, I should be writing a french tutorial on using UMEP and UWG for UHI simulation. It will be some kind of documentation for my company, and I hope it’ll be of use.

**Comment by biglimp** [comment]
See here for explanation on grid extent (Search throughout the grid extent) [here](https://umep-docs.readthedocs.io/en/latest/pre-processor/Urban%20Morphology%20Morphometric%20Calculator%20(Grid).html#urban-morphology-morphometric-calculator-grid). Wind direction is not considered but could be if you were to use the anisotrophic output from Land Cover Fraction and Morphometric calculator. However, then you manually need to prepare your data for the wind direction you want to consider.

UWG is a local scale model and I would not recommend that go lower than e.g. 400 times 400 meters as the processes will not be valid below those grid extents.

The Analyzer considers only night time. An update is coming soon that clarifies this.

Great with more tutorials! that is always welcome.

**Reply by PetitMote** [reply]
Thanks for the answer!

I’ve read the explanation, but I can’t understand the impact on the model.

Ouch, 400m is quite a large scale to study urban effects. I don’t know if urban parks would be visible at that scale :/
